src/constants.telemetry.ts

```diff
@@ -324,6 +324,13 @@ export type TelemetryEvents = {
 		connected: boolean;
 		type: StartWorkType;
 	};
+	/** Sent when the user takes an action on a StartWork issue */
+	'startWork/issue/action': StartWorkEventData & {
+		action: 'soft-open';
```

Comment:
This should be typed similar to `startWork/issue/chosen` below (add `connected` and `type` properties, and either leave `item.` flexible as is done there, or update that event with tighter typing on `item.` properties), with the addition of `action`.